poster.go

```diff
@@ -30,7 +30,7 @@ func (st AnalysisStatus) String() string {
 // Poster can post comments about an event.
 type Poster interface {
 	// Post posts comments about an event.
-	Post(context.Context, Event, []*Comment) error
+	Post(context.Context, Event, []AnalyzerComments) error
```

Comment:
The problem with that is that the feedback url does not apply to all the comments. The comments may come from several different analyzers, each one with a different feedback url.

Reply:
I understand.
Actually, it is one of the reasons why I prefer `Post(context.Context, Event, []*Comment, *PostOptions)`. We can post comments independently for each analyzer. Then we would not need for loops inside for loops everywhere, but only one in `server.post`. And we may also post them as soon as analyzer replied without always waiting for the last one if we want.

Reply:
That also makes sense.
But it implies a UX change that we may or may not do:

Comments are posted grouped inside the same pull request review. If we do this change, we would post a new review for each analyzer.
This would make more sense if each review came from a different user name. But if all of them come from the 'lookout' account, it might be strange.